package.json

```diff
@@ -11,10 +11,16 @@
     "url": "git+https://github.com/ngParty/Angular1-scaffold.git"
   },
   "scripts": {
-    "typings": "typings install",
-    "postinstall": "npm run typings",
-    "start": "webpack-dev-server --port $npm_package_config_port --hot --inline --progress --profile --colors --watch --display-error-details --display-cached --content-base src/",
-    "build": "NODE_ENV=production webpack --config webpack-build.config.js"
+    "rimraf": "rimraf",
+    "typings": "typings",
+    "postinstall": "npm run typings -- install",
+    "start": "npm run server",
+    "build": "NODE_ENV=production webpack --config webpack-build.config.js",
+    "build:dev": "webpack --progress --profile --colors --display-error-details --display-cached",
```

Comment:
Yeah I have seen such error already, not sure why but you are trying to write to root directory on your computer. Check config, it will be something with NODE_ENV